### ISSUE TITLE
Volk patch already applied upstream

### ISSUE THREAD

**Issue by carlesfernandez** [balister/meta-sdr#54]
Hi @balister , 

the patch at https://github.com/balister/meta-sdr/blob/dunfell/recipes-support/volk/files/0001-Check-for-lib64-verus-lib-and-set-LIB_SUFFIX-accordi.patch has been already applied upstream in volk 2.4.1. I think this patch should be removed from the volk recipe, both in master and dunfell branches.

Currently, I'm getting the following warning:

```
WARNING: volk-2.4.1-r0 do_patch: Fuzz detected:

Applying patch 0001-Check-for-lib64-verus-lib-and-set-LIB_SUFFIX-accordi.patch
patching file CMakeLists.txt
Hunk #1 succeeded at 230 with fuzz 1 (offset 105 lines).

The context lines in the patches can be updated with devtool:

    devtool modify volk
    devtool finish --force-patch-refresh volk <layer_path>

Don't forget to review changes done by devtool!

WARNING: volk-2.4.1-r0 do_patch: QA Issue: Patch log indicates that patches do not apply cleanly. [patch-fuzz]
```

Thanks!

**Comment by balister** [comment]
You are correct. I had it stuck in my head that the patch hadn't made 2.4.1 and the fuzz was due to another patch moving this one slightly. I'll good ahead and fix the 2.4.1 recipes. Thanks for reminding me!

**Comment by balister** [comment]
Dunfell and master are updated. Thanks for the help reminding me!
